docs/developer/how-to/make-new-ophyd-async-device.md

```diff
@@ -0,0 +1,58 @@
+# Decision Flowchart for Creating a New ophyd_async Device
+
+This document contains decision flowcharts designed to guide developers through the process of creating a new ophyd_async device in the Ophyd library. These flowcharts help in determining the appropriate class inheritance, methods to override, and testing procedures for optimal device functionality and integration into the system.
+
+## High-Level Development Flowchart
+
+This high-level flowchart guides the overall process of creating a new ophyd_async device, from testing PVs to overriding methods and determining state enums.
+
+```{mermaid}
```

Comment:
```suggestion
```mermaid
```

Should get this to render?

Reply:
idk, for me it renders in vscode preview